plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4704,7 +4704,7 @@ protected void verifyIcebergTableProperties(MaterializedResult actual)
         assertThat(actual).isNotNull();
         MaterializedResult expected = resultBuilder(getSession())
                 .row("write.format.default", format.name())
-                .build();
+                .row("write.parquet.compression-codec", "zstd").build();
```

Comment:
@nastra this doesn't seem right - the parquet compression codec property is relevant only for parquet files (not for avro, orc). 
https://github.com/trinodb/trino/issues/20401

Reply:
we should adress this as part of https://github.com/apache/iceberg/issues/9490